### DISCUSSION TITLE
Most simple way to play anim montage on upper body only?

### DISCUSSION THREAD

**Discussion by deafstoryteller** [Sixze/ALS-Refactored#439] [Q&A] [answered]
For playing an anim montage on only upper body, I see some posts about using post locomotion plus curves and others using layers, then some recent ones on tag blending- curious what the most modern solution is?

Would love screenshots and/or links if possible and thank you!

**Comment by MrWilsonWilson** [comment, marked as answer]
Most simple is to to a layered blend per bones, and make the chain from the spine of your choosing. 
Most modern I'm not sure, but the complex way of post locomotion + curves gives quite a lot of control.
Sorry for no screenshots, you can easily do research on layered blend per bones and get some insight !

**Reply by deafstoryteller** [reply]
Thank you!!! I'm starting to understand the post locomotion+ curves but am still a bit lost on which curves I need for turning the leg animations off.

Been trying the legs/ layer legs /layering legs / foot lock etc 

The examples I've been trying to learn from is high mantle and also get up front. (If there's better examples or als documentation on curves lmk!)

Is the als get up front the opposite of what I need to do? (Instead of 1/0 foot left ik I would start with 0/1 correct?)

**Reply by deafstoryteller** [reply]
[Best](https://www.youtube.com/watch?v=ja-C9VcNyKw&t=181) video I found on bone blend- https://www.youtube.com/watch?v=ja-C9VcNyKw&t=181

It's working beautifully now, thank you so much kind madam or sir.
